cmd/plakar/subcommands/exec/exec.go

```diff
@@ -90,7 +90,7 @@ func cmd_exec(ctx *appcontext.AppContext, repo *repository.Repository, args []st
 	}
 	go func() {
 		defer stdout.Close()
-		io.Copy(os.Stdout, stdout)
+		_, _ = io.Copy(os.Stdout, stdout)
```

Comment:
don't wont to sound rude, but I hate this kind of things.  IMHO these constructs are less readable than the implicit discarding of the returned value.

(same for all the other places where this change was done -- I'm skeptical this really improves the quality of the codebase and/or avoids bugs.)

Reply:
See https://github.com/PlakarKorp/plakar/pull/317#issuecomment-2619062380